darkmoon/cli/main.py

```diff
@@ -39,7 +39,7 @@ def get_metadata(
         ),
     ],
 ) -> None:
-    """Call all of the metadata functions and send data to api endpoint."""
+    """Call the get metadata function from utils.py and send data to api endpoint."""
```

Comment:
For all of these, you don't need to say exactly what is run. The docstring should give a general purpose, parameters, raises (exceptions), and returns